### PR TITLE
Postprocessing block enablement with DPK fuzzy dedup 

### PR DESCRIPTION
**Pull request by yuanchi2807**

### CODE REVIEW

File: fms_dgt/base/task.py
[the remark this answers is not in the document]
Comment (sivasankalpp) [reply fms_dgt/base/task.py:1]: yeah, I think @yuanchi2807 a rebase should fix this?

[the remark this answers is not in the document]
Comment (yuanchi2807) [reply fms_dgt/base/task.py:1]: Certainly. Need to trace up how to toggle the file format switch.

File: fms_dgt/base/block.py [hunk 1]
```diff
@@ -156,3 +156,28 @@ def generate(
...
+        for x in inputs:
+            inp_args, inp_kwargs = self.get_args_kwargs(x, arg_fields, kwarg_fields)
+            res = self._validate(*inp_args, **inp_kwargs)
+            if res:
+                self.write_result(x, res, result_field)
+                outputs.append(x)
+        return outputs
+
+    @abstractmethod
+    def _validate(self, *args: Any, **kwargs: Any) -> bool:
```
[the remark this answers is not in the document]
Comment (yuanchi2807) [reply fms_dgt/base/block.py:182]: Thanks. Exactly the feedback I am looking for! 
However, from the yaml structure, the differences among various types of blocks do not really jump out. What are the guidelines to apply which block type when?

**Review by mvcrouse — APPROVED** [review]
Looks good to me!

**Review by sivasankalpp — APPROVED** [review]

**Pull request merged**